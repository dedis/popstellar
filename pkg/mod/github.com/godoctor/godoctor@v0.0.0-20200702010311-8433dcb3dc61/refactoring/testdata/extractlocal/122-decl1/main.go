package main

import "fmt"

func main() {
	x := []int{1 + 2, 4, 5} //<<<<< var,6,13,6,17,newVar,pass
	fmt.Println(x)
}
