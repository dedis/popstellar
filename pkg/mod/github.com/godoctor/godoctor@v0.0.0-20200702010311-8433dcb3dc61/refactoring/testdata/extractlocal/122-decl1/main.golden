package main

import "fmt"

func main() {
	newVar := 1 + 2
	x := []int{newVar, 4, 5} //<<<<< var,6,13,6,17,newVar,pass
	fmt.Println(x)
}
